Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.World.cpp

```diff
@@ -1764,6 +1764,263 @@ int CLuaFunctionDefs::GetBirdsEnabled(lua_State* luaVM)
     return 1;
 }
 
+int CLuaFunctionDefs::RemoveBirds(lua_State* luaVM)
+{
+    if (CStaticFunctionDefinitions::RemoveBirds())
+    {
+        lua_pushboolean(luaVM, true);
+        return 1;
+    }
+    lua_pushboolean(luaVM, false);
+    return 1;
+}
+
+int CLuaFunctionDefs::AddBirds(lua_State* luaVM)
+{
+    //  bool addBirds ( float startX, float startY, float startZ, float destX, float destY, float destZ, [int birdsCount = 1, int birdType = 0, bool checkObstacles = false] )
+
+    CVector          vecStartPosition, vecDestPosition;
+    int              iNumBirds, iBirdType;
+    bool             bCheckObstacles;
+    CScriptArgReader argStream(luaVM);
+
+    argStream.ReadVector3D(vecStartPosition);
+    argStream.ReadVector3D(vecDestPosition);
+    argStream.ReadNumber(iNumBirds, 1);
+    argStream.ReadNumber(iBirdType, 0);
+    argStream.ReadBool(bCheckObstacles, false);
+
+    if (!argStream.HasErrors())
```

Comment:
These errors should be changed accordingly (https://github.com/multitheftauto/mtasa-blue/issues/821#issuecomment-522187565).

Reply:
Is this an example of how to do it right?

Before
```cpp
    if (!argStream.HasErrors())
    {
        lua_pushboolean(luaVM, g_pGame->GetTimeCycle()->SetTimeCycle(strPropertyName, fValue, iWeatherID, iCycleNum));
        return 1;
    }
    else
        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());

    lua_pushboolean(luaVM, false);
    return 1;
```

After
```cpp
    if (argStream.HasErrors())
    {
        return luaL_error(luaVM, argStream.GetFullErrorMessage());
    }
    else
        lua_pushboolean(luaVM, g_pGame->GetTimeCycle()->SetTimeCycle(strPropertyName, fValue, iWeatherID, iCycleNum));
        return 1;

    lua_pushboolean(luaVM, false);
    return 1;
```

Reply:
```cpp
    if (argStream.HasErrors())
    {
        return luaL_error(luaVM, argStream.GetFullErrorMessage());
    }

    lua_pushboolean(luaVM, g_pGame->GetTimeCycle()->SetTimeCycle(strPropertyName, fValue, iWeatherID, iCycleNum));
    return 1;
```

Is probably what you want.